ctapipe/image/hillas.py

```diff
@@ -118,20 +118,25 @@ def hillas_parameters_1(geom: CameraGeometry, image):
     phi = np.arctan2(mean_y, mean_x)
 
     # Higher order moments
-    sk = cos_delta * (pix_x - mean_x) + sin_delta * (pix_y - mean_y)
+    if length == 0:
+        skewness = np.nan
```

Comment:
it may be better to simply init these to nan at the top, not in an *if* branch, since then you need another unit test to test this branch (hence why codecov is complaining about this change).  Same for other places you added if-statements. 

Also comparing length to exactly 0 may not be a good idea, as it's a floating point quantity.